src/pages/AdminDashboard/CreateRoomForm/CreateRoomForm.tsx

```diff
@@ -77,7 +77,9 @@ const CreateRoomForm = ({ fields, onFormSubmit, onCreateFormReset }) => {
         event: React.MouseEvent<HTMLElement>,
         newValue: string | null
     ) => {
-        setToggleValue(newValue)
+        if(newValue !== null) {
+            setToggleValue(newValue)
```

Comment:
why is this needed, considering the state has already an initial value ?

Reply:
When you unselect a button, the value returned is null. In fact it disable the possibility to unselect a button.

ref : https://material-ui.com/components/toggle-button/#enforce-value-set